### PR TITLE
CSM-11439: Added get permission for aws lambda code signing config

### PR DESCRIPTION
**Pull request by rgurav-uptycs**
Updated the terraform script by adding the new aws lambda permission to fetch the lambda function code signing config

### CODE REVIEW

File: lambdas/constants.js [hunk 1]
```diff
@@ -42,7 +42,9 @@ const ReadOnlyPolicy = `{
               "sns:GetSubscriptionAttributes",
               "ssm:ListCommandInvocations",
               "ce:GetCostAndUsage",
-              "redshift-serverless:List*"
+              "redshift-serverless:List*",
+              "lambda:GetCodeSigningConfig",
+			  "lambda:GetFunctionCodeSigningConfig"
```
Comment (llukalapu-uptycs) [inline lambdas/constants.js:47]: @rgurav-uptycs we need to update zip file 
and keep it in proper format

Comment (rgurav-uptycs) [reply lambdas/constants.js:47]: @llukalapu-uptycs How we can do that?

Comment (rgurav-uptycs) [reply lambdas/constants.js]: @llukalapu-uptycs fixed the formatting

Comment (rgurav-uptycs) [reply lambdas/constants.js]: @llukalapu-uptycs How can we update the zip file

Comment (rgurav-uptycs) [reply lambdas/constants.js]: Done.

**Review by llukalapu-uptycs — APPROVED** [review]

**Pull request merged**